src/parser/MutateSentences.h

```diff
@@ -473,6 +473,67 @@ class DeleteEdgeSentence final : public Sentence {
     std::unique_ptr<WhereClause>                whereClause_;
 };
 
+class DownloadSentence final : public Sentence {
+public:
+    DownloadSentence() {
+        kind_ = Kind::kDownload;
+    }
+
+    const std::string* host() const {
+        return host_.get();
+    }
+
+    void setHost(std::string *host) {
+        host_.reset(host);
+    }
+
+    const int32_t port() const {
+        return port_;
+    }
+
+    void setPort(int32_t port) {
+        port_ = port;
+    }
+
+    const std::string* path() const {
+        return path_.get();
+    }
+
+    void setPath(std::string *path) {
+        path_.reset(path);
+    }
+
+    const std::string* localPath() const {
+        return localPath_.get();
+    }
+
+    void setLocalPath(std::string *localPath) {
+        localPath_.reset(localPath);
+    }
+
+    void setUrl(std::string *url) {
+        std::string u = url->substr(7, url->size());
```

Comment:
Add some comments for the magic number.

Reply:
maybe can use macro to replace the magic number.